browser/search_engines/search_engine_tracker.h

```diff
@@ -33,6 +34,19 @@ enum class SearchEngineP3A {
   kMaxValue = kBrave,
 };
 
+enum class SearchEngineSwitchP3A {
+  kNoSwitch,
```

Comment:
there are nine buckets and AFAIK we are limited with 8 by P3A design

Reply:
Indeed. I've asked for feedback on that.

Reply:
I've cut it down to 8 responses, removing the overlap with 'default is brave' per discussion with @solso.